packages/common/src/ceramic-api.ts

```diff
@@ -135,6 +135,10 @@ export interface CeramicApi extends CeramicSigner {
 }
 
 export interface MultiQuery {
+  /**
+   * The genesis content for the queried stream
```

Comment:
```suggestion
   * The genesis content for the queried stream. Useful in cases where the stream might not exist and you want to avoid timing out trying to load the genesis commit from IPFS.
```